From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Parker Wahle <parkeredwardwahle2017@gmail.com>
Date: Thu, 30 Dec 2021 14:18:05 -0500
Subject: [PATCH] bump l4j


diff --git a/build.gradle.kts b/build.gradle.kts
index 4cb69d06139756b765156f9df86ed3fefdeb2417..94ba12c6e51e082b4dcd497eb49dbd2f7f07cc05 100644
--- a/build.gradle.kts
+++ b/build.gradle.kts
@@ -46,11 +46,11 @@ dependencies {
           all its classes to check if they are plugins.
           Scanning takes about 1-2 seconds so adding this speeds up the server start.
      */
-    implementation("org.apache.logging.log4j:log4j-core:2.14.1") // Paper - implementation
-    annotationProcessor("org.apache.logging.log4j:log4j-core:2.14.1") // Paper - Needed to generate meta for out hex color converter plugin
+    implementation("org.apache.logging.log4j:log4j-core:2.17.1") // Paper - implementation
+    annotationProcessor("org.apache.logging.log4j:log4j-core:2.17.1") // Paper - Needed to generate meta for out hex color converter plugin
     // Paper end
-    implementation("org.apache.logging.log4j:log4j-iostreams:2.14.1") // Paper
-    implementation("org.apache.logging.log4j:log4j-api:2.14.1") // Paper
+    implementation("org.apache.logging.log4j:log4j-iostreams:2.17.1") // Paper
+    implementation("org.apache.logging.log4j:log4j-api:2.17.1") // Paper
     implementation("org.ow2.asm:asm:9.1")
     implementation("org.ow2.asm:asm-commons:9.1") // Paper - ASM event executor generation
     implementation("com.googlecode.json-simple:json-simple:1.1.1") {
